backend/src/channels/channels.service.spec.ts

```diff
@@ -49,8 +54,13 @@ describe('ChannelsService', () => {
 
   describe('when finding all Channels', () => {
     it('should return array of Channel', async () => {
+      jest
+        .spyOn(repository, 'findAndCount')
+        .mockReturnValue(Promise.resolve([[new Channel()], 1]));
       const ret = await service.findAll();
-      expect(ret.data.length).toBe(2);
+      expect(ret).toBeInstanceOf(PageDto);
+      expect(ret.data.length).toBe(1);
+      expect(ret.data[0]).toBeInstanceOf(Channel);
```

Comment:
```suggestion
      const ret = await service.findAll();
      expect(ret.data.length).toBe(2);
      expect(ret).toBeInstanceOf(PageDto);
      expect(ret.data[0]).toBeInstanceOf(Channel);
```
This would work with the acceptance of my suggestion on `repository.mock.ts`.